### PR TITLE
Add Cache-Control max-age variable to publish_to_s3 script

### PR DESCRIPTION
**Pull request by riiwo**
This is needed to have some assets with shorter life span.

EN-5028

### CODE REVIEW

File: vars/deployer.groovy [hunk 1]
```diff
@@ -105,7 +105,8 @@ private def inToolbox(Map args = [:], Closure body) {
 
 def publishAssets(Map args) {
   def defaultArgs = [
-    s3Bucket: 'libs.salemove.com'
+    s3Bucket: 'libs.salemove.com',
+    cacheMaxAge: 31536000
```
Comment (wifilinker) [inline vars/deployer.groovy]: seconds?

Comment (riiwo) [reply vars/deployer.groovy]: yes

Comment (riiwo) [reply vars/deployer.groovy]: https://github.com/salemove/pipeline-lib/pull/87/commits/043147fa7692fde48a5b946b709240f39649ffcc

**Comment by riiwo** [comment]
!squash

**Comment by riiwo** [comment]
!deploy

**Comment by deiwin** [comment]
There's docs as well for this, that need to be updated: https://github.com/salemove/pipeline-lib#publishassets

**Comment by sm-deployer** [comment]
Deploy failed or was aborted. @riiwo, please check the logs [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-87/4/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-87/4//console)).

**Comment by riiwo** [comment]
!merge

**Comment by deiwin** [comment]
!check

**Pull request merged**